shared/linechart.js

```diff
@@ -97,7 +98,13 @@ function renderLineChart(elem, data, width, height, options) {
         })
         .attr("x", 40)
         .attr("dy", "0.35em")
+        .attr("fill", function (d) {
```

Comment:
```suggestion
        .attr("class", "line-chart-label")
        .style("fill", function (d) {
```
1st line reasoning:
* we can keep the styling of the line chart labels in the sass files
* this won't override the font used for the labelling
2nd line: this will override the css (otherwise the css in the sass files take precedence over simple attributes)